package.json

```diff
@@ -50,5 +50,8 @@
       "packages/**"
     ]
   },
+  "resolutions": {
+    "superagent": "^10.2.0"
```

Comment:
This overrides superagent for `panoptes-client`, while that still uses superagent 8.
- https://github.com/zooniverse/panoptes-javascript-client/issues/251